client/tailscale/localclient.go

```diff
@@ -391,16 +391,26 @@ func (lc *LocalClient) DebugAction(ctx context.Context, action string) error {
 	return nil
 }
 
+// DebugPortmapOpts contains options for the DebugPortmap command.
+type DebugPortmapOpts struct {
+	// LogHTTP instructs the debug-portmap endpoint to print all HTTP
+	// requests and responses made to the logs.
+	LogHTTP bool
+}
+
 // DebugPortmap invokes the debug-portmap endpoint, and returns an
 // io.ReadCloser that can be used to read the logs that are printed during this
 // process.
-func (lc *LocalClient) DebugPortmap(ctx context.Context, duration time.Duration, ty, gwSelf string) (io.ReadCloser, error) {
+func (lc *LocalClient) DebugPortmap(ctx context.Context, duration time.Duration, ty, gwSelf string, opts *DebugPortmapOpts) (io.ReadCloser, error) {
```

Comment:
normally we don't change the signature of things in this package because it's kinda a stable API, but this is named "Debug" so I'll let it slide :)

Reply:
Yeah, I figured that things named `Debug` are not in our stability guarantee 😅